### PR TITLE
Moved the "DepdendencyInjection" repo from mvc to kcore area.

### PR DESCRIPTION
**Pull request by mkArtakMSFT**
@Eilon , can you please have a look at this.

### CODE REVIEW

File: repoSets.json [hunk 1]
```diff
@@ -95,6 +95,11 @@
         "repo": "DataProtection",
         "inclusionLevel": "AllItems"
       },
+      {
+        "org": "aspnet",
+        "repo": "DependencyInjection",
```
Comment (Eilon) [inline repoSets.json]: The repo is already listed in the `kcore` section, but it's listed as  ItemsAssignedToPersonSet (or whatever). So, all that needs to happen is to change it from ItemsAssigned to AllItems.

File: repoSets.json [hunk 2]
```diff
@@ -570,11 +575,6 @@
         "repo": "CORS",
         "inclusionLevel": "AllItems"
       },
-      {
-        "org": "aspnet",
-        "repo": "DependencyInjection",
-        "inclusionLevel": "AllItems"
```
Comment (Eilon) [inline repoSets.json]: Similar to the above comment, we don't want to remove the repo entirely, but instead just flip it from AllItems to ItemsAssigned.

File: repoSets.json (hunk 1 above)
Comment (mkArtakMSFT) [reply repoSets.json]: I see, thanks !

**Comment by Eilon** [comment]
Will be live in ~10min.

cc @muratg FYI

**Pull request merged**